build/shade/_k-standard-goals.shade

```diff
@@ -151,24 +150,11 @@ default SAMPLES_PROJECT_GLOB = "samples/*/*.csproj"
               }
           }
       });
-
-      projectGlobs.AddRange(srcProjects);
-    }
-    if (!BuildSrcOnly && Directory.Exists("test"))
-    {
-        projectGlobs.AddRange(Files.Include(TEST_PROJECT_GLOB).ToList());
-    }
-    if (!BuildSrcOnly && Directory.Exists("samples") && Directory.EnumerateFiles("samples", "*.csproj", SearchOption.AllDirectories).Any())
-    {
-        projectGlobs.AddRange(Files.Include(SAMPLES_PROJECT_GLOB).ToList());
     }
 
-    if (projectGlobs.Any())
+    foreach (var sln in Files.Include(Path.Combine(BASE_DIR, "*.sln")))
```

Comment:
That's true. I guess we should get Mvc to move the second sln (the NoFun one) out of the repo root as part of the migration. We'll switch to calling `dotnet build` and have it fail in the non-common cases.

FYI @NTaylorMullen